packages/server/src/util/test-utils/roster.test-generator.ts

```diff
@@ -0,0 +1,39 @@
+import { CustomRosterColumn } from '../../api/roster/custom-roster-column.model';
+import { Unit } from '../../api/unit/unit.model';
+import { Roster } from '../../api/roster/roster.model';
+import { uniqueEdipi } from './unique';
+import { getCustomRosterColumnTestData } from './custom-roster-column.test-generator';
+import { getOrgTestData } from './org-test-generator';
+import { getAdminUserTestData } from './user.test-generator';
+import { getUnitsTestData } from './unit.test-generator';
+import { getReportSchemaTestData } from './report-schema.test-generator';
+
+export function getRosterTestData(numRosterEntries: number, customColumn: CustomRosterColumn, units: Unit[], numUnits: number) {
+  const rosterEntries: Roster[] = [];
+  for (let i = 0; i < numRosterEntries; i++) {
+    const customColumns: any = {};
+    customColumns[customColumn.name] = `custom column value`;
```

Comment:
Just a thought but we should change the the values here so better tests can be written for checking individual custom columns.

Reply:
This is copy/paste from seed.ts. This request is out of the scope of this PR I think.